src/exampleTypes.js

```diff
@@ -44,6 +44,26 @@ export default F.stampKey('type', {
       getValues: x => _.map('name', x.context.options),
     },
   },
+  dateRangeFacet: {
+    label: 'List',
+    validate: validateValues,
+    reactors: {
+      values: 'others',
+      mode: 'others',
+      size: 'self',
+      optionsFilter: 'self',
```

Comment:
I don't think most of these are on your type - you'll also probably want to add a reaction for `ranges`

Reply:
Changes done. It might be better than to have a new type in CR without all these UI elements since they are not supported by the type and you would need to configure the facet to remove them every time ...